doc/conf.py

```diff
@@ -5,11 +5,8 @@
 
 
 _g = globals()
-_g.update(
-    build_package_configs(
-        project_name="ts_CBP", version=lsst.ts.cbp.version.__version__
-    )
-)
-
-intersphinx_mapping["ts_xml"] = ("https://ts-xml.lsst.io", None)
-intersphinx_mapping["ts_salobj"] = ("https://ts-salobj.lsst.io", None)
+_g.update(build_package_configs(project_name="ts_CBP", version=lsst.ts.cbp.__version__))
+
+intersphinx_mapping["ts_xml"] = ("https://ts-xml.lsst.io", None)  # noqa
+intersphinx_mapping["salobj"] = ("https://ts-salobj.lsst.io", None)  # noqa
+intersphinx_mapping["simactuators"] = ("https://ts-simactuators.lsst.io", None)  # noqa
```

Comment:
Why did you remove the ts_ prefix from salobj and omit it from simactuator (while leaving it for ts_xml)? It presumably works either way, but I'm curious. I have been using the package name as the key (ts_xml, ts_salobj...).

Reply:
I'm still curious about this. It's an odd inconsistency.